lib/livebook_web/live/hub/file_system_form_component.ex

```diff
@@ -88,7 +88,13 @@ defmodule LivebookWeb.Hub.FileSystemFormComponent do
             </p>
           <% end %>
           <div class="flex space-x-2">
-            <.button type="submit" disabled={not @changeset.valid?}>
+            <.button
+              type="submit"
+              disabled={
+                not @changeset.valid? or
+                  match?(%Livebook.Hubs.Team{active: false}, @hub)
```

Comment:
Any reason to not pass `@disabled` from upstream, as in the other ones?

Reply:
good call, done in a6aad6ad, thanks!